### ISSUE TITLE
EXO 2.0.4 Transcripts being cluttered with import cmdlet message

### ISSUE THREAD

**Issue by anthonielance** [MicrosoftDocs/office-docs-powershell#7343]
Transcript logs are being cluttered with useless `VERBOSE: Importing cmdlet 'Test-ActiveToken'` message. The cmdlet is constantly being logged and providing no diagnostic or troubleshooting help. It would be nice if this message can be suppressed or better yet importing the cmdlet only once instead of every time a exposed cmdlet such as `Add-DistributionGroupMember` is called.

Example Code

```powershell
Start-Transcript -Path "C:\Logs\DistribtuionList.txt" -Append
$VerbosePreference = "Continue"

....

Write-Verbose "Adding Marking Distribution List members"
Get-AdUsers -Filter {Department -eq 'Marketing'} -Properties Mail |
ForEach-Object {
  Add-DistributionGroupMember -Identity 'marketing@comp.com' -Member $_.Mail
  Write-Verbose "Added '$($_.Mail)' to marketing list"
}

....

$VerbosePreference = "SilentlyContinue"
End-Transcript
```

Example Log

```log
....

VERBOSE: Adding Marketing Distribution List members
VERBOSE: Importing cmdlet 'Test-ActiveToken'.
VERBOSE: Added 'JSmith@comp.com' to marketing list
VERBOSE: Importing cmdlet 'Test-ActiveToken'.
VERBOSE: Added 'KJones@comp.com' to marketing list
VERBOSE: Importing cmdlet 'Test-ActiveToken'.
VERBOSE: Added 'CTester@comp.com' to marketing list
VERBOSE: Importing cmdlet 'Test-ActiveToken'.
VERBOSE: Added 'LJenkins@comp.com' to marketing list

....
```

**Comment by dariomws** [comment]
@anthonielance - thank you for engaging with the community here for the docs.

I know the product teams are always looking for user feedback and how to make EXO PS module better. They keep a close tab on user feedback, and it would be awesome if you could add your voice in User Voice https://office365.uservoice.com/forums/273493-office-365-admin?category_id=96338 or [blog](https://techcommunity.microsoft.com/t5/exchange-team-blog/exo-v2-powershell-module-is-now-generally-available-on-linux-amp/ba-p/2121811).

You might find others have made a similar request that you can add to or open a new one. Let us know here if you do so we can up-vote your request.

Here we are limited to documentation issues and improvements.

Thank you for your contribution to make the docs better! Much appreciated!

**Comment by yogkumgit** [comment]
@anthonielance  Thank you very much for your kind feedback. Considering your valuable feedback, as advised please log this issue as a User Voice. Concerning team will evaluate and take the appropriate action, as per their schedule. We proceed here to close it. Thanks again for taking out some time and sharing your feedback.
